test/__snapshots__/invalid-options.test.js.snap

```diff
@@ -3,9 +3,9 @@
 exports[`errors 1`] = `
 Array [
   "Error: main.0c220ec66316af2c1b24.js from UglifyJs
-DefaultsError: \`invalid-option\` is not a supported option",
+\`invalid-option\` is not a supported option",
   "Error: manifest.6afe1bc6685e9ab36c1c.js from UglifyJs
-DefaultsError: \`invalid-option\` is not a supported option",
+\`invalid-option\` is not a supported option",
```

Comment:
I fixed the BUG: `src / index.js> buildError ()`. So the results are not the same as before:

```js
    } else if (err.msg) {
      return new Error(`${file} from UglifyJs\n${err.msg}`);
    }
    return new Error(`${file} from UglifyJs\n${err.stack}`); // -> ...DefaultsError: `invalid-option` is not a supported option"
```

```js
    } else if (err.message) {
      return new Error(`${file} from UglifyJs\n${err.message}`); // -> ...`invalid-option` is not a supported option"
    }
    return new Error(`${file} from UglifyJs\n${err.stack}`);
```